src/collective/fingerpointing/logger.py

```diff
@@ -12,33 +14,53 @@
 FORMAT = '%(asctime)s - %(levelname)s - %(message)s'
 
 # by default, the audit log will use the same location used for the event log
-eventlog = getattr(getConfiguration(), 'eventlog', None)
+zopeConf = getConfiguration()
+
+auditlogConf = getattr(
+    zopeConf,
+    'product_config',
+    {}
+).get(
+    'collective.fingerpointing',
+    {}
+)
 
-if eventlog is None:
-    # we are running tests
-    logfile = os.path.join('.', AUDITLOG)
-else:
-    try:
-        logpath = eventlog.handler_factories[0].instance.baseFilename
-        logfolder = os.path.split(logpath)[0]
-        logfile = os.path.join(logfolder, AUDITLOG)
-    except AttributeError:
-        # we are in the debug console
+logfile = auditlogConf.get('audit-log', None)
+if logfile is None:
```

Comment:
@pcdummy as discussed before, we're not guessing anymore the location of the `audit.log` file; if no location is given, then no additional `audit.log` file must be created and we only log to `event.log`.